app/services/twilio_message.rb

```diff
@@ -11,15 +11,7 @@ def self.activate(number)
     client.messages.create(
       from: Figaro.env.twilio_phone_number,
       to: number,
-      body: "Welcome to Kackle!"
-    )
-  end
-
-  def self.deactivate(number)
-    client.messages.create(
-      from: Figaro.env.twilio_phone_number,
-      to: number,
-      body: "Successfully unsubscribed."
+      body: "Welcome to Kackle! Respond with 'no' if you believe you received this message in error."
```

Comment:
so if they respond with no and we deactivate them, are we activating them automatically? sorry if i'm missing something here!